extractor/src/main/java/org/schabi/newpipe/extractor/services/soundcloud/extractors/SoundcloudStreamExtractor.java

```diff
@@ -185,43 +187,102 @@ public String getHlsUrl() {
         try {
             final JsonArray transcodings = track.getObject("media").getArray("transcodings");
 
-            // get information about what stream formats are available
-            for (Object transcoding : transcodings) {
+            // Iterate a first time to see if there is a progressive MP3 stream available.
+            // If yes, the MP3 HLS stream will be not added to audioStreams.
 
+            boolean mp3ProgressiveStreamInTranscodings = false;
+
+            for (final Object transcoding : transcodings) {
+                final JsonObject t = (JsonObject) transcoding;
```

Comment:
No single letter variables

Reply:
It was initially this variable name so I kept it. I will change it.

Reply:
I changed `t` to `transcodingJsonObject`.